internal/view/index.html

```diff
@@ -171,6 +173,19 @@
 			}
 		})
 	</script>
+	<script type="module">
+		if ('serviceWorker' in navigator) {
+			window.addEventListener('load', function() {
+				navigator.serviceWorker.register('pwabuilder-sw.js');
+			});
+		}
+	</script>
+	<script type="module">
+		import 'https://cdn.jsdelivr.net/npm/@pwabuilder/pwaupdate';
```

Comment:
Can we have this living with our other static assets? It seems weird to me that we serve everything bundled withing the binary but this is fetched from an outside service.